src/main/java/com/cloudbees/AWSCredentialsUtils.java

```diff
@@ -0,0 +1,60 @@
+package com.cloudbees.jenkins.plugins.awscredentials;
```

Comment:
The class is not in the correct directory to match the package. Please relocate the class, keep this `package` declaration.

Reply:
That's so weird. I did test this, not sure how it managed to compile. O_O